packages/dashboard/package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.3.1-unstable.7",
   "private": true,
   "scripts": {
-    "predev:docker": "docker-compose -f ./docker-compose.dev.yml up --detach --remove-orphans",
-    "dev:docker": "next dev",
-    "dev": "next dev",
+    "predev:docker": "docker-compose -f ./docker-compose.build.yml build",
+    "dev:docker": "docker-compose -f ./docker-compose.run.yml --env-file .env.ganache up",
```

Comment:
make this command name match the env file it is using.

Reply:
Updated PR